### ISSUE TITLE
Training discriminator

### ISSUE THREAD

**Issue by Dawars** [MandyMo/pytorch_HMR#15]
I have been able to get everything working except the discriminator.

How do you train the discriminator? As far as I can tell, the code only loads the trained model.

**Comment by Dawars** [comment]
On a second look, the discriminator learns alongside the generator, I'll close the issue

**Comment by hsyntemiz** [comment]
Why don't we backpropagate loss of discriminator first? Is this a mistake? 

```
            e_opt.zero_grad()
            e_loss.backward()
            e_opt.step()

            d_opt.zero_grad()
            d_loss.backward()
            d_opt.step()
```
